src/Test/Utilities/Portable/Compilation/OperationTreeVerifier.cs

```diff
@@ -726,7 +726,14 @@ public override void VisitUnaryOperatorExpression(IUnaryOperatorExpression opera
             LogString(nameof(IUnaryOperatorExpression));
 
             var kindStr = $"{nameof(UnaryOperationKind)}.{operation.UnaryOperationKind}";
-            LogString($" ({kindStr})");
+            if (operation.IsLifted)
+            {
+                LogString($" ({kindStr}-IsLifted)");
```

Comment:
This is not the convention we've been going with for properties so far, and I also don't like how it looks. I'd prefer `({kindStr}, IsLifted)`